css/main.css

```diff
@@ -1,3 +1,663 @@
+  align-items: center;
+  justify-content: center;
+  height: 200px;
+}
+
+/* footer */
+
+footer {
+  background-color: var(--dark-green);
+  color: var(--white);
+  text-align: left;
+}
+
+footer h1 {
+  color:  var(--black);
+  text-align: center;
+}
+
+.follow-us {
+  background-color: var(--white);
+  height: 180px;
+  padding-top: 2rem;
+}
+
+footer img {
+  position: relative;
+  width: 131px;
+  height: 159px;
+  object-fit: cover;
+  object-position: 50% 50%;
+}
+
+.image-container {
+  height: 250px;
+
+}
+
+.img-1 {
+  transform: rotate(-20deg);  
+  left: 75px;
+  top: 70px;
+}
+
+.img-2 {
+  transform: rotate(20deg);
+  right: 100px;
+  bottom: 20px;
+}
+
+.img-3 {
+  transform: rotate(-20deg);
+  left: 90px;
+  bottom: 270px;
+  }
+
+.img-4 {
+  transform: rotate(20deg);
+  left: 10px;
+  bottom: 170px;
+  }
+
+.footer-nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 25px;  
+}
+
+ul, label{
+  font-family:"HelveticaNeueW01-45Ligh";
+  list-style-type:none;
+  padding-left: 0;
+  line-height: 2;
+  font-weight: 50;  
+}
+
+.footer-email {
+  padding: 1.5rem;
+}
+
+.email-submit {
+  display: flex;
+  flex-direction: row;
+  align-items: end;
+  margin: 2rem 0;
+  bottom: 0;
+}
+
+.submit-btn {
+  background-color: transparent;
+  border:2px solid var(--white);
+  width: 160px;  
+}
+
+.email-submit input {
+  width: 120px;
+  bottom: 0;
+  border: none;
+  border-bottom: 2px solid var(--white) ;
+  background-color: transparent;
+}
+
+.juice-bar-logo img{
+  width: 80px;
+  height: auto;
+}
+
+.footer-bottom {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 0 1.3rem;
+  height: 200px;
+}
+
+.footer-bottom .footer-note{
+  line-height: 1.1;
+  font-size: 14px;
+  margin: 0;
+  text-align: left;
+}
+
+a {
+  color: var(--white);
+}
+
+/* breakpoint 2 --> 900px */
+
+@media screen and (min-width: 900px) {
+ 
+
+  .header { 
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    height: 88px;
+    background-color: var(--white);
+    position: fixed;
+    width: 100%;
+  }
+
+  .login {
+    display:flex; 
+    width: 120px;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .login img {
+    height: 50px;
+  }
+
+  .header p {
+    margin:0;
+  }
+
+  .header-top {
+    display: flex;
+    justify-content: space-between;
+    width: 30%;
+  }
+
+  .header-bottom {
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    width: 50%;
+    padding: 2rem;
+  }
+
+  .header .order-btn {
+    height: 88px;
+    width: 200px;
+  }
+
+  .header .order-btn:hover{
+    background-color: var(--medium-green);
+    transition: 0.8s;
+  }
+
+  h3 {
+    font-size: 35px;
+  }
+
+  .main-top h3 {
+    font-size: 24px;
+  }
+
+  h1 {
+    font-size: 55px;
+  }
+
+  p {
+    font-size: 18px;
+  }
+
+  .main-grid {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-column: 1 / -1;
```

Comment:
well done for using `grid` and `grid-column`. Do you think you need 12 columns? It looks like most of your rows span all columns anyway as you have applied `grid-column: 1 / -1;`

Reply:
You are right, Ellie. I change it.